### DISCUSSION TITLE
Best practicies/advices/tips

### DISCUSSION THREAD

**Discussion by rtritto** [mongo-express/mongo-express#972] [Ideas]
### PRs
1. **Use Fix keyword to autoclose issue**
   - *How to*: in the start comment of PR, use `Fix #<PR_number_of_issue>` to attach the PR to issue
   - *Result*: after the PR is closed, automatically the issue will be triggered and closed
2. **Use "Squash and merge" when merging**
   - *How to*: before to merge a PR, select `"Squash and merge"`, choose if edit the message and merge
   - *Result*: on the target branch, the result is only 1 commit (cleaner history)

### Commits
1. **Use [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0) notation in the commit message**

**Comment by rtritto** [comment]
FYI @shakaran @BlackthornYugen

**Reply by rtritto** [reply]
Updated title and content

FYI @shakaran @BlackthornYugen

**Comment by shakaran** [comment]
Ideally, we should start to use "[Conventional Commits notation](https://www.conventionalcommits.org/en/v1.0.0/)"

In commit title if you include the `fix #<PR_number_of_issue>` should work too to close the issue related when merged.

**Comment by rtritto** [comment]
FYI @xTudoS
